nservicebus/outbox/index.md

```diff
@@ -41,7 +45,7 @@ Note: On the wire level the Outbox guarantees `at-least-once` message delivery,
 
 ## Important design considerations
 
- * The business data and deduplication data need to be stored in the same database.
+ * Business data and Outbox deduplication data must be stored in the same database, in order to take advantage of the same local database transaction to store all data.
```

Comment:
I would make this a warning then. 

Also, pretty sure that sentence doesn't need a comma!